xarray_jax/__init__.py

```diff
@@ -109,20 +109,6 @@ def foo(input: xarray.Dataset) -> xarray.Dataset:
 import xarray
 
 
-# Types which we wrap with JaxArrayWrapper to allow creating xarray datatypes
-# from them.
-# Note this includes some non-Array types which jax sometimes needs to use as
-# leaves of pytrees, in order to ensure we can still use xarray datatypes as
-# internal pytree nodes in these cases.
-_WRAPPED_TYPES = (
-    jax.Array, jax.ShapeDtypeStruct, jax.stages.ArgInfo, jax.stages.OutInfo)
-
-
-def Variable(dims, data, **kwargs) -> xarray.Variable:  # pylint:disable=invalid-name
```

Comment:
Please leave
`Variable = xarray.Variable`
commented as a deprecated alias for backwards compatibility.